README.md

```diff
@@ -2,4 +2,8 @@
 
 ## Entity Relationship Diagram
 
-[![](https://mermaid.ink/img/eyJjb2RlIjoiZXJEaWFncmFtXG4gICAgICAgIFVTRVIgIHx8LS18eyBDT01NSVQgOiBwdXNoZXNcbiAgICAgICAgQ09NTUlUICB9by0tb3sgQ09NTUlULURJQUdOT1NJUyA6IGhhc1xuXG4iLCJtZXJtYWlkIjp7InRoZW1lIjoiZGVmYXVsdCJ9LCJ1cGRhdGVFZGl0b3IiOmZhbHNlfQ)](https://mermaid-js.github.io/mermaid-live-editor/#/edit/eyJjb2RlIjoiZXJEaWFncmFtXG4gICAgICAgIFVTRVIgIHx8LS18eyBDT01NSVQgOiBwdXNoZXNcbiAgICAgICAgQ09NTUlUICB9by0tb3sgQ09NTUlULURJQUdOT1NJUyA6IGhhc1xuXG4iLCJtZXJtYWlkIjp7InRoZW1lIjoiZGVmYXVsdCJ9LCJ1cGRhdGVFZGl0b3IiOmZhbHNlfQ)
+[![](docs/er-diagram.png)](https://mermaid-js.github.io/mermaid-live-editor/#/edit/eyJjb2RlIjoiZXJEaWFncmFtXG4gICAgICAgIFVTRVIgIHx8LS18eyBDT01NSVQgOiBwdXNoZXNcbiAgICAgICAgQ09NTUlUICB9by0tb3sgQ09NTUlULURJQUdOT1NJUyA6IGhhc1xuXG4iLCJtZXJtYWlkIjp7InRoZW1lIjoiZGVmYXVsdCJ9fQ)
+
+## Data-flow Diagram
+
+[![](docs/dfd.jpeg)](https://app.lucidchart.com/invitations/accept/3e43c9ee-b220-4501-97f3-f23a11a8261e)
```

Comment:
It seems lucidchart requires registration. Sorry for the extra work, but can you move the diagram to use draw.io or mermaidjs instead?

Reply:
sure